packages/core/src/config/config.ts

```diff
@@ -193,7 +193,7 @@ export type Network = {
    * }
    * ```
    */
-  transport: Transport;
+  transport: Transport | unknown;
```

Comment:
What did this do?

Reply:
Solves a very common problem with mismatched types in viem. `transport` is a unique case because the user is often importing viem, then using `http()` and passing it to the config.